src/centralstore/incremental/NodeBlock.h

```diff
@@ -0,0 +1,73 @@
+/**
+Copyright 2020 JasminGraph Team
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+**/
+
+#include <cstring>
+#include <fstream>
+#include <iostream>  // for cout
```

Comment:
I think we are not using cout in these sources anymore. Hence can we remove the line `#include <iostream>`

Reply:
Done